modules/firehose/module.go

```diff
@@ -14,6 +14,9 @@ import (
 
 const (
 	releaseConfigString     = "release_configs"
+	replicasString          = "replicas"
+	RUNNING                 = "RUNNING"
+	STOPPED                 = "STOPPED"
```

Comment:
are these constants going to be used outside this package? If not, they should be private. They can probably be renamed to `releaseStateRunning`, `releaseStateStopped`, etc.

Reply:
Yes, sounds good! Let's do this.